hr_payroll_extended/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Payroll Commissions and Reinbursements
+=========================
+
+This module allows you to add commission and reimbursement to payslip
+
+** Features list :**
+    * Add Worked Hours to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add Holiday Rate to contract.
+    * Add commision rate, commision and reimbursement to contract.
+

```

Comment:
AFAICS there is more than that (link to expenses and invoices). Can you create a simple usage section and a use case?